### PR TITLE
Update respec, remove terms

### PR DESCRIPTION
**Pull request by jnurthen**
* remove terms
* update respec profile
* add xrefs

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/mathml-aam/pull/8.html" title="Last updated on Nov 8, 2021, 5:04 PM UTC (a1e4b69)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mathml-aam/8/bbfbe02...a1e4b69.html" title="Last updated on Nov 8, 2021, 5:04 PM UTC (a1e4b69)">Diff</a>

### CODE REVIEW

**Review by joanmarie — APPROVED** [review]
Thanks for doing this!

**Comment by fred-wang** [comment]
I'm doing a subset of this in https://github.com/w3c/mathml-aam/pull/22

File: index.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <meta charset="utf-8" />
     <title>MathML Accessiblity API Mappings 1.0</title>
     <script src="https://www.w3.org/scripts/jquery/1.11.3/jquery.min.js"></script>
-    <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove" defer="defer"></script>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer="defer"></script>
     <script src="common/script/resolveReferences.js" class="remove"></script>
     <script src="common/biblio.js" class="remove" defer="defer"></script>
-
+    <script src="common/script/mapping-tables.js"></script>
```
Comment (fred-wang) [inline index.html:10]: why do we need to move this inclusion here in this PR?

File: index.html [hunk 2]
```diff
@@ -23,10 +23,7 @@
 	  github: "w3c/mathml-aam",
 	  copyrightStart:  "2020",
 
-	  wg: "Accessible Rich Internet Applications Working Group",
-	  wgURI: "https://www.w3.org/WAI/ARIA/",
-	  wgPublicList: "public-aria",
```
Comment (fred-wang) [inline index.html:28]: wgPublicList is not deprecated: https://respec.org/docs/#wgPublicList

Is group: "aria" enough to deduce the public mailing list? It didn't seem to be the case when I tested.

File: index.html [hunk 3]
```diff
@@ -53,13 +50,14 @@
 	      "PR" : "https://www.w3.org/TR/wai-aria-1.1/",
 	      "REC": "https://www.w3.org/TR/wai-aria/"
 	  },
-
-	  preProcess: [ linkCrossReferences ]
+	    preProcess:[linkCrossReferences],
+		postProcess:[],
+		definitionMap:[],
+		xref: ["core-aam", "accname", "wai-aria"]
```
Comment (fred-wang) [inline index.html:56]: can you explain why this is needed in this PR?

File: index.html [hunk 4]
```diff
@@ -145,11 +143,6 @@ <h3>Deprecated</h3>
       </section>
     </section>
 
-    <section id="glossary">
-      <h2>Important Terms</h2>
-      <div data-include="common/terms.html" data-oninclude="restrictReferences"></div>
-    </section>
-
```
Comment (fred-wang) [inline index.html:152]: That seems a separate thing, but I'm curious where it was discussed to remove the terms?
(i agree it's maybe not important to have them here and probably that duplicates content from other spec)

File: index.html (hunk 3 above)
Comment (jnurthen) [reply index.html:56]: - linkCrossReferences deals with links between different ARIA specs (rref, pref etc.) and resolves them correctly
- xref deals will look in those other specs for references and allows the terms to be referenced correctly.

File: index.html (hunk 4 above)
Comment (jnurthen) [reply index.html:152]: - Other (non-aria) specs need to reference some of these terms
- if they are defined in multiple places which do they reference?
- xrefs now deals with references to terms in other specs

File: index.html (hunk 2 above)
Comment (jnurthen) [reply index.html:28]: I prefer to direct feedback through github so don't like the links at the top. There are still links in the SOTD section for anyone that needs the mailing list. 
Editors decision if you want this I believe.

File: index.html (hunk 1 above)
Comment (jnurthen) [reply index.html:10]: It stopped it breaking.

**Pull request merged**